src/Type/DataSchema.ts

```diff
@@ -0,0 +1,53 @@
+import SchemaProperty from './SchemaProperty';
+
+/**
+ * Class represents the schema of imported geo-data, to have information about available
+ * properties and data ranges.
+ * Comparable to a DescribeFeatureType response for an OGC WFS.
+ * This is modelled as JSON schema:
+ *
+ *  {
+ *    "title": "Person",
+ *    "type": "object",
+ *    "properties": {
+ *      "firstName": {
+ *        "type": "string"
+ *      },
+ *      "lastName": {
+ *        "type": "string"
+ *      },
+ *      "age": {
+ *        "description": "Age in years",
+ *        "type": "integer",
+ *        "minimum": 0
+ *      }
+ *   }
+ * }
+ *
+ * @type DataSchema
+ */
+type DataSchema = {
+
+  /**
+   * Optional title for the described entity
+   *
+   * @type {string}
+   */
+  title: string | '';
+
+  /**
+   * Optional type definition for the described entity
+   *
+   * @type {string}
+   */
+  type: string | 'object';
```

Comment:
I think it should be `type?: string;` if its optional. `'object'` allready is a string.

Reply:
By this notation I wanted to ensure that ``type`` is always a String (= default value) and not ``null`` or ``undefined``.  Does this make sense?

Reply:
see above